daprdocs/content/en/operations/observability/tracing/otel-collector/open-telemetry-collector.md

```diff
@@ -6,42 +6,48 @@ weight: 900
 description: "How to use Dapr to push trace events through the OpenTelemetry Collector."
 ---
 
-{{% alert title="Note" color="primary" %}}
-Dapr directly writes traces using the OpenTelemetry (OTEL) protocol as the recommended method. For observability tools that support OTEL protocol, you do not need to use the OpenTelemetry Collector.
+Dapr directly writes traces using the OpenTelemetry (OTEL) protocol as the **recommended** method. For observability tools that support OTEL protocol, you don't need to use the OpenTelemetry Collector.
```

Comment:
@msfussell is the [current diagram in the doc](https://docs.dapr.io/operations/observability/tracing/otel-collector/open-telemetry-collector/) ok to use, or does that need to be updated? 
![image](https://github.com/dapr/docs/assets/94493363/993359b3-ce4e-4c2a-91bf-715c4911a523)

Reply:
@msfussell @hhunter-ms IMO the diagram is still correct but maybe the verbiage could be updated to something like:

"Dapr directly writes traces using the OpenTelemetry (OTEL) protocol as the **recommended** method. For observability tools that support OTEL protocol, it is still recommended to use the OpenTelemetry Collector since it allows your app to offload data quickly and includes other features like retries, batching and encryption. For more information, read the Open Telemetry [documentation](https://opentelemetry.io/docs/collector/#when-to-use-a-collector)."

Reply:
thanks @alicejgibbons! I updated the doc with your revised intro